src/scene/text/TextStyle.ts

```diff
@@ -130,6 +131,13 @@ export interface TextStyleOptions
     wordWrap?: boolean;
     /** The width at which text will wrap, it needs wordWrap to be set to true */
     wordWrapWidth?: number;
+    /**
+     * An optional filter or array of filters to apply to the text, allowing for advanced visual effects.
+     * These filters will be applied to the text as it is created, resulting in faster rendering for static text
+     * compared to applying the filter directly to the text object (which would be applied at run time).
+     * @default null
```

Comment:
The default value would be `undefined` right?

Reply:
yes!